src/modules/projects/components/tables/ProjectHouseholdsTable.tsx

```diff
@@ -106,14 +106,20 @@ export const HOUSEHOLD_COLUMNS: {
     header: 'Relationship to HoH',
     render: (hh) => (
       <TableCellContainer>
-        {hh.householdClients.map((c) => (
-          <HmisEnum
-            key={c.id}
-            value={c.relationshipToHoH}
-            enumMap={HmisEnums.RelationshipToHoH}
-            whiteSpace='nowrap'
-          />
-        ))}
+        {hh.householdClients.map((c) =>
```

Comment:
@gigxz could you give more info about why you reverted this? I'm seeing missing "Data Not Collected" info on the table:
![Screenshot 2025-01-10 at 2 31 45 PM](https://github.com/user-attachments/assets/41fa2c4f-1328-4908-b2cc-247cb2692e01)

Edit: It looks like this would be resolved in the Tables deploy branch, anyway: https://github.com/greenriver/hmis-frontend/pull/1020/files#diff-819ec0ace67cab35cca989ef6e3f3d5a6187dec26e6dcbcb1d4090614788a6c5R43-R53

Reply:
I didn't change the current behavior here, enrollments with null/99 relationship to hoh still appear like they do in your screenshot. On a previous version I had removed DataNotCollected from the Enum entirely, which is why you see that commit history, but I reverted that change.